README.md

```diff
@@ -1,2 +1,24 @@
-# images-videos-colling-edem
-HTML project that matches the appearance of Youtube.
+# Clone of Youtube Page
+
+---
+## Authors
+### This is a microverse project by Edem and Collins
+You can find more about Edem [here](https://github.com/edemagbenyo) and collins [here](https://github.com/collinsugwu).
+
+## Tech stack
+HTML, CSS
+In this project we are building a web page that match the appearance of [Youtube.com](http://archive.today/Bss88) signup page.
+
+---
+
+## Setup
+Here is how to setup this project on your machine.
+
+Open your terminal in your working folder and execute the following command
+
+>git clone https://github.com/edemagbenyo/images-videos-collins-edem
+
+
+Here is the link to the live version of this project	
+[here](https://rawcdn.githack.com/edemagbenyo/images-videos-collins-edem/8749c5c20fd665623673327e159868e8a6f34916/index.html)
+
```

Comment:
#### `OPTIONAL`

Nice job adding a descriptive readme.

Make it even more professional by adding the assignment link (from Odin),  screenshots of the page and what was learned during the project.